provider-ui/ui_dev_server.js

```diff
@@ -12,11 +12,9 @@ const httpProxy = require('http-proxy');
 
 const proxy = httpProxy.createProxyServer({});
 
-proxy.on('error', (res) => {
-  res.writeHead(500, {
-    'Content-Type' : 'text/plain',
-  });
-  res.end('Proxy issue via provider-ui.');
+proxy.on("error", function (err, req, res) {
+  res.writeHead(500, { "Content-Type" : "text/plain" });
+  res.end('Proxy issue vi Meshery Provider-UI');
```

Comment:
I wonder if this is intended to be "via".

Reply:
Yes supposed to be via, but I copied and pasted this from `ui`, so it must be spelled wrong there too.